layers/meta-balena-imx8mplus/recipes-kernel/linux/linux-compulab_%.bbappend

```diff
@@ -25,14 +23,6 @@ BALENA_CONFIGS[cf80211] = " \
 	CONFIG_CFG80211=m \
 "
 
-# We don't load grub with u-boot, instead
-# balenaOS u-boot loads the kernel directly
-BALENA_CONFIGS:append = " noefi "
-BALENA_CONFIGS[noefi] = " \
-	CONFIG_EFI_STUB=n \
-	CONFIG_EFI=n \
-"
-
```

Comment:
Could these configs be defaulted from meta-balena in a new distro feature of some sort that boards would include when sb is needed?

Reply:
hey @floion these are not for secure boot support, this commit just adds the balena bootloader after U-Boot.

So would it make sense to have a `balena-bootloader` distro feature that would add EFI support to the kernel? Maybe, but up until now all BSPs have defined EFI and all this commit does is remove the device specific removal of EFI support. So even if there was a EFI setting in the kernel class this would still be needed I reckon.

I won't add it just now, but I'll keep it in mind for future work.

Reply:
My bad, I meant to say balena bootloader, sb was just in my head and I typed it in :)